kubernetes.go

```diff
@@ -309,6 +322,10 @@ type kubernetesNodePoolsRoot struct {
 	Links     *Links                `json:"links,omitempty"`
 }
 
+type kubernetesUpgradesRoot struct {
+	AvailableUpgradeVersions []*KubernetesVersion `json:"available_upgrade_versions,omitempty"`
```

Comment:
Small nit, we references versions as `VersionSlug`s (https://github.com/digitalocean/godo/pull/229/files#diff-d1bf2f66114f0ed852c9c697d6083ee1R54) should we do the same here?

Reply:
These aren't slugs - they're full version objects that include both the slug and the k8s version. I did include `Slug` in the [`KubernetesClusterUpgradeRequest`](https://github.com/digitalocean/godo/pull/229/files#diff-d1bf2f66114f0ed852c9c697d6083ee1R74) since it is just a slug there.